modules/client/src/main/java/org/apache/ignite/internal/client/table/ClientKeyValueView.java

```diff
@@ -544,6 +554,45 @@ public CompletableFuture<Void> streamData(Publisher<Entry<K, V>> publisher, @Nul
         return ClientDataStreamer.streamData(publisher, opts, batchSender, provider, tbl);
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public CompletableFuture<AsyncCursor<Entry<K, V>>> queryAsync(
+            @Nullable Transaction tx,
+            @Nullable Criteria criteria,
+            CriteriaQueryOptions opts
+    ) {
+        return tbl.getLatestSchema()
```

Comment:
As I see most part of the method the same for few realization and difference just for mapping. Can we extract common part and pass just a mapping part as parameter?